packages/xstate-store/src/types.ts

```diff
@@ -365,14 +367,40 @@ export interface Readable<T> extends Subscribable<T> {
   get: () => T;
 }
 
-export interface Atom<T> extends Subscribable<T>, Readable<T> {
+export interface BaseAtom<T> extends Subscribable<T>, Readable<T> {}
+
+export interface InternalBaseAtom<T> extends Subscribable<T>, Readable<T> {
```

Comment:
Those `Internal*` types are leaking as part of the public exports of the package right now because we just blindly reexport everything contained in this file. Ideally this would get fixed